bower.json

```diff
@@ -28,7 +28,7 @@
     "selectize": "0.12.1",
     "c3": "0.4.11",
     "kubernetes-label-selector": "2.1.1",
-    "kubernetes-container-terminal": "2.1.1",
+    "kubernetes-container-terminal": "3.0.1",
```

Comment:
Were there any other changes needed to go from xterm 2.x to xterm 3.x?

Reply:
Good catch.  Yup.  https://github.com/openshift/origin-web-console/commit/aff114e73d86c02a3db09e3bfc49cd5c2d140498  Lemme update.

Reply:
Updated.  Thanks, @spadgett.